pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -273,3 +273,8 @@ type BuildRetention struct {
 func init() {
 	SchemeBuilder.Register(&Build{}, &BuildList{})
 }
+
+// GetLabels returns the labels of the Build
+func (b Build) GetLabels() map[string]string {
+	return b.Labels
+}
```

Comment:
I don't think this is needed because Kubernetes already provides that function on `ObjectMeta`.